### ISSUE TITLE
Chance of an Apk?

### ISSUE THREAD

**Issue by Thekiller2498** [eak65/Cashapp#4]
I've struggled trying to comprehend the instructions to run this on android let alone running this at all. Could u make an android version or apk or something like that?

**Comment by KrxpyKrxme** [comment]
Yes please lmk if this works just message my discord if it works @chadperone#0430

**Comment by Keyfbe** [comment]
- [ ] I'm on iPhone trying to figure out how to download this can someone please help me

**Comment by Zrlee06** [comment]
I would really really like an apk. I am on android and I am having lots of problems trying to run this.

**Comment by proxinitty** [comment]
> Yes please lmk if this works just message my discord if it works @chadperone#0430

Can you help me get it on iphone? I don't know how to do it.

**Comment by Psyclapped** [comment]
For everybody asking, [here you go.](https://github.com/eak65/Cashapp/issues/11)
